src/module-config/module-config.ts

```diff
@@ -80,6 +85,23 @@ function getConfigForModule(moduleName: string): ConfigObject {
             }
           }
         }
+        if (schema[key].arrayElements && schema[key].arrayElements.validators) {
+          if (!Array.isArray(value)) {
+            throw Error(
+              `Invalid configuration value ${value} for ${thisKeyPath}: value must be an array.`
+            );
+          }
+          for (let validator of schema[key].arrayElements.validators) {
+            for (let element of value) {
+              const validatorResult = validator(element);
```

Comment:
could you provide an example signature of a validator function?

Reply:
Yeah, good idea! I've updated the docs: https://wiki.openmrs.org/display/projects/openmrs-esm-module-config

A validator function should be created with the `validator` function, e.g.
```
validator(n => /\d/.test(n), must contain a number)
```